spec/services/hyrax/collection_types/create_service_spec.rb

```diff
@@ -1,8 +1,20 @@
 RSpec.describe Hyrax::CollectionTypes::CreateService do
-  describe ".create_collection_type" do
-    it 'creates a default collection type when no options are received' do
-      described_class.create_collection_type
-      expect(Hyrax::CollectionType).to exist(machine_id: Hyrax::CollectionType::USER_COLLECTION_MACHINE_ID)
+  describe '.create_collection_type' do
+    it 'create collection type with default options when no options are received' do # rubocop:disable RSpec/ExampleLength
+      described_class.create_collection_type(machine_id: 'custom_type', title: 'Custom Type')
+      ct = Hyrax::CollectionType.find_by_machine_id('custom_type')
+      expect(ct.machine_id).to eq('custom_type')
+      expect(ct.title).to eq('Custom Type')
+      expect(ct.description).to eq('')
+      expect(ct.nestable).to be true
+      expect(ct.discoverable).to be true
+      expect(ct.sharable).to be true
```

Comment:
Could this be `expect(ct).to be_nestable`, `expect(ct).to be_discoverable`, `expect(ct).to be_sharable`?

Reply:
This is easy enough to do, but you either end up with some tests in the suggested format, e.g. `expect(ct).to be_nestable`, and some in the existing format, e.g.  `expect(ct.assigns_workflow).to be true`
OR
you end up with statements that do not read well if they are all converted, e.g. `expect(ct).not_to be_assigns_workflow`

Does this kind of refactor buy us an advantage that justifies spending time on it?